### ISSUE TITLE
System-Error while Menuhax is booting

### ISSUE THREAD

**Issue by YoungKyoko** [yellows8/3ds_homemenuhax#160]
I tried to run menuhax v3.0 on my n3ds 11.0.0-33E and I got no error or problem while installing whatsoever. But when I try to run MenuHax (using the L Button) my Screen turns after showing the splash image black. On the bottom screen it says that an error occured: 
http://puu.sh/qr5dx/cbf3794c5f.jpg

ik its german but what it basically says it that an error occured, that I should turn my system off and report it to nintendos support if the error occures more than ones. I tried to boot menuhax a few times always with the same result.

any ideas what I could do to fix it?

**Comment by yellows8** [comment]
https://github.com/yellows8/3ds_homemenuhax/issues/131#issuecomment-237367023

**Comment by DragonPhx** [comment]
I have the same thing :/

**Comment by ghost** [comment]
@DragonPhx have you read yellows8's post?

**Comment by DragonPhx** [comment]
Yes, but me it is about problem during startup and not during installation .. As Youngkyoko

**Comment by yellows8** [comment]
......

**Comment by PrometheusG** [comment]
I had the same problem and I don't remember exactly what I did But it got itself fixed somehow and now I don't see the bottm screen"an error has occured" etc etc when launching menuhax.

Firstly, I skiped ropbin-payload setup as @yellows8  said.it got installed successfully but I had the error screen issue and couldn't boot to menuhax.

Then I tried to deleted menuhax, but I got an error code(sorry I don't remember :-(  ).Tried to install it again, this time by pressing B and an error showed up again.I booted to my 3ds,I deleted the paid theme that I used and choose the default theme.rebooted again,downloaded the theme I deleted and swapped it changing the default theme.

Lastly, I choose to install menuhax again and this time by pressing A and again an error showed up.I reconfigured menuhax (type 1) and pressed start to exit.I rebooted my console by pressing the corresponding key and it worked.

I don't know how, but that is all I did.
I hope it helped!
